charter.md

```diff
@@ -0,0 +1,210 @@
+# observe-k8s
+
+## Mission Statement
+
+For cloud native workloads, observability is a key requirement, yet it’s a broad term and there are many tools and approaches available to achieve it.
+
+While the [whitepaper](https://github.com/cncf/tag-observability/blob/main/whitepaper.md) by the [CNCF TAG Observability](https://github.com/cncf/tag-observability) covers the theoretical foundations of modern observability, practitioners need guidance on how to achieve it in real life scenarios.
+
+This working group aims to provide:
+
+- a suite of realistic sample applications, which employ observability tools & techniques to demonstrate how practitioners can gain insight and understanding.
+- actionable, good practice & peer reviewed guidance on how to enable observability for real-life workloads. Observed workloads will be CNCF Projects.
+
+_A distinct section requiring intentional navigation by the User shall be provided for open source projects which are not CNCF Projects if this opportunity arises._
+
+
+## WG (Working Group) Requirements for Launch
+
+```
+    TAGs may choose to spawn focused and time-limited working groups to achieve some of their responsibilities (for example, to produce a specific educational white paper, or portfolio gap analysis report). Working groups should have a clearly documented charter, timeline (typically a few quarters at most), and set of deliverables. Once the timeline has elapsed, or the deliverables delivered, the working group dissolves, or is explicitly re-chartered.
+```
+
+    (source: https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#responsibilities--empowerment-of-tags)
+
+- Charter / Description of WG (_This document once it's merged_)
+- Time Box:
+  - 3 quarters (Q1-Q3 2022)
+- Deliverables:
+  - github.com/observe-k8s: An umbrella organization for repos that realize the goals of this org
```

Comment:
I wasn't sure if "org" should be "working group"? It felt self circular otherwise

Reply:
I interpreted "org" as being short for "this GitHub organization" (`observe-k8s`).

Reply:
That's how I read it, but it didn't make sense to me.

Are the goals of the working group the same goals as the GH org, or are they different? Or is it to allow for the working group to foster the creation of the GH org to live its own life? So initially the goals are aligned but over time, perhaps once the working group is disbanded, the GH org goals will move on and evolve?